pages/apim/3.x/user-guide/publisher/service-discovery/service-discovery-eureka.adoc

```diff
@@ -4,4 +4,71 @@
 :page-layout: apim3x
 :page-title: Eureka service discovery
 
-include::https://raw.githubusercontent.com/gravitee-io/gravitee-service-discovery-eureka/master/README.adoc[]
+== Configuration
+
+The configuration is loaded from the common GraviteeIO Gateway configuration file (gravitee.yml) All configurations described by official Eureka discovery client documentation are available on the official github repository: https://github.com/Netflix/eureka/wiki. Gravitee Gateway uses Eureka Discovery client v1 and fetch configured Eureka Servers. It does not register the gateway application into Eureka registry. Please refer the official Eureka documentation for advanced uses and advanced concepts.
+
+For more information, see the complete link:{{ 'https://github.com/gravitee-io-community/gravitee-service-discovery-eureka/blob/master/README.adoc' }}[Eureka Client Config].
+
+== Enable Eureka Service Discovery in Gravitee.io APIM
+
+Now that your configured your Eureka service, go to APIM console:
+
```

Comment:
These steps are part of a sequence so should be numbered llist bullets - please replace all asterisks (* ) with dots (. ) or numbers at the start of each step.